report/cucumber_report.json

```diff
@@ -0,0 +1,156 @@
+[
```

Comment:
this should be ignored - everything from report folder should not be pushed, add it to .gitignore file